### PR TITLE
Add network options to NetworkInfo to display internal network option

### PR DESCRIPTION
**Pull request by chenchun**
Signed-off-by: Chun Chen <ramichen@tencent.com>

### CODE REVIEW

**Comment by chenchun** [comment]
If we want to display the option in `docker network inspect`, we need to add `NetworkOptions` in `NetworkInfo`. Related to https://github.com/docker/docker/pull/18926

File: network.go [hunk 1]
```diff
@@ -61,6 +61,7 @@ type NetworkInfo interface {
 	IpamConfig() (string, []*IpamConf, []*IpamConf)
 	IpamInfo() ([]*IpamInfo, []*IpamInfo)
 	DriverOptions() map[string]string
+	NetworkOptions() map[string]string
```
Comment (aboch) [inline network.go]: Libnetwork options are well-known options being set via NetworkOption setter functions and not via a map[string]string like opaque driver options.
Also, as I commented on the docker PR, we cannot assume future network option can nicely fit in a  string value. 

I'd rather have an explicit structure containing the network options or a getter function for each of the options. Regarding this PR, something like
`Internal() bool`

**Comment by mavenugo** [comment]
LGTM

**Comment by aboch** [comment]
Nice, 1 line change, isn't that better ?! ;-)

LGTM

**Comment by chenchun** [comment]
Yea :laughing:

**Pull request merged**